### PR TITLE
chore: align style use between docs and demo

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1.  Clean up how we are calling different stylesheets across the use cases.   Ensure's tokens are included in demos and docs as a layer in the main stylesheets for both in a modular approach.  Now a single source of truth for both use cases which is more cleanly organized and can be extended using layers.

## Testing Instructions

1. View in localhost:8000 and doc demos localhost:8080 demos should now have the same styles across both environments.

### CODE REVIEW

**Comment by bennypowers** [comment]
Im ok with global tokens style sheets in the demos, but nothing more

Users should be able to copy demo contents onto a blank page and get results

**Comment by zeroedin** [comment]
> Users should be able to copy demo contents onto a blank page and get results

This changes nothing about that, this is more just house keeping on our end.

File: docs/styles/demo/styles.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
   typography,
   base,
   layout,
   third-party;
 
 @import url('../reset.css') layer(reset);
 @import url('../fonts.css') layer(fonts);
+@import url('/assets/packages/@rhds/tokens/css/global.css') layer(tokens);
 @import url('../typography.css') layer(typography);
 @import url('../third-party/prism-rhds.css') layer(third-party);
```
Comment (zeroedin) [inline docs/styles/demo/styles.css:14]: @bennypowers I can't remember why we loaded prism into the demo styles.  This maybe something we can remove, especially with changes to codeblock, can you verify?

**Comment by zeroedin** [comment]
> Im ok with global tokens style sheets in the demos, but nothing more

@bennypowers PTAL a look again, this last commit might have clarified stuff a bit more.

**Comment by zeroedin** [comment]
Just realized this the part where i run into the dev-server trying to call global.css from the node_modules and gets wrapped in js...

File: .pfe.config.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "description": "Red Hat Design System",
     "componentSubpath": "elements",
     "stylesheets": [
-      "/docs/styles/demo/styles.css"
+      "/docs/styles/demo/dev-server.css"
```
Comment (bennypowers) [inline .pfe.config.json:19]: We can lose patch on pfe this way, right?

**Review by bennypowers — APPROVED** [review]

File: .pfe.config.json (hunk 2 above)
Comment (zeroedin) [reply .pfe.config.json:19]: I think I'd like to break this out to another issue.  IMHO I'd still like to make this configurable upstream, both the html and the CSS.  We should be able to control/override all the CSS we want downstream.

Comment (zeroedin) [reply .pfe.config.json:19]: https://github.com/patternfly/patternfly-elements/issues/2846

**Pull request merged**